blog/_posts/2017-11-30-bloop-release.md

```diff
@@ -0,0 +1,158 @@
+---
+layout: blog-detail
+post-type: blog
+by: Jorge Vicente Cantero and Martin Duhem
+title: "Meet bloop, a fast tool to compile and test your project"
+---
+
+Martin Duhem and I are happy to announce `bloop`, a tool that Scala developers
+can use to compile and test their projects faster without being tied to their
+stock build tool.
```

Comment:
Maybe make this even clearer? It's the compile-test-run loop. Right now, the description still sounds like a build tool because "compiling and testing" is what you do with a build tool. Can you keep on working to make this more specific? It'd also help to say what bloop stands for here; "build loop," right? That might also make it clearer to people what this tool actually is.

Reply:
I still think that you should refer to the tweet that started this whole project, no?

Reply:
> It'd also help to say what bloop stands for here; "build loop," right? That might also make it clearer to people what this tool actually is.

Good point!